views/editor/bramble.html

```diff
@@ -117,10 +117,13 @@
 
     <div id="spinner-container">
         <div class="thimble-spinner"></div>
-        <div class="loading-message message">{{ gettext("loadingProject") }}</div>
+        <div class="loading-message message">{{ gettext("loadingProject") }}
+	<p>{{ gettext("loading problem") }} <button type="button"><a href="https://support.mozilla.org/products/webmaker/thimble">{{ gettext("get help here") }}</button></a></p>
```

Comment:
Let's keep it simple and remove the button, let's just make it a simple anchor and we can style it if needed.